parsec/core/logged_core.py

```diff
@@ -40,6 +45,55 @@
 
 logger = get_logger()
 
+FAILSAFE_PATTERN_FILTER = re.compile(
+    r"^\b$"
+)  # Matches nothing (https://stackoverflow.com/a/2302992/2846140)
+
+
+def _get_pattern_filter(pattern_filter_path: Path) -> Optional[Pattern]:
+    try:
+        data = pattern_filter_path.read_text()
+    except OSError as exc:
+        logger.warning(
+            f"Path to the file containing the filename patterns "
+            f"to ignore is not properly defined: {exc}"
+        )
+        return None
+    try:
+        regex = "|".join(
+            fnmatch.translate(line.strip())
+            for line in data.splitlines()
+            if line.strip() and not line.strip().startswith("#")
+        )
+    except ValueError as exc:
+        logger.warning(
+            f"Could not parse the file containing the filename patterns " f"to ignore: {exc}"
```

Comment:
`" f"` due to black

and shouldn't we use `exc_info=exc` to pass the exception ?

Reply:
Good catch!